cpp/include/Ice/Communicator.h

```diff
@@ -138,9 +138,7 @@ namespace Ice
         /// It is legal to create an object adapter with the empty string as its name. Such an object adapter is
         /// accessible via bidirectional connections or by collocated invocations.
         /// @param name The object adapter name.
-        /// @param serverAuthenticationOptions The %SSL configuration properties for server connections.
-        /// The `SSL::ServerAuthenticationOptions` type is an alias to the platform specific %SSL server authentication
-        /// options.
+        /// @param serverAuthenticationOptions The SSL configuration properties for server connections.
```

Comment:
maybe we should say options instead of properties here.

Reply:
Yes. I replaced all "configuration properties" by "options" when talking about SSL options.

I'd rather reserve the term configuration to Ice properties / config files etc. Not include options classes/structs.